lib/irb/command/copy.rb

```diff
@@ -39,8 +39,12 @@ def execute(arg)
       private
 
       def copy_to_clipboard(text)
-        IO.popen(clipboard_program, 'w') do |io|
-          io.write(text)
+        if Gem.win_platform?
+          Kernel.system("powershell.exe", "-NoProfile", "-Command", "Set-Clipboard", "-Value", text)
```

Comment:
> On Windows, passing text via stdin does not work reliably in IRB

This might be fixed in https://github.com/ruby/reline/pull/875
Although I also think adding this workaround for a while is worth. What do you think?